src/ng2-smart-table/components/cell/cell-types/cell-textarea.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Output, EventEmitter, Input } from '@angular/core';
+
+import { Cell } from '../../../lib/data-set/cell';
+import { DefaultCellType } from './default-cell-type';
+
+@Component({
+  selector: 'cell-type-textarea',
+  template: `
+    <textarea [ngClass]="inputClass"
+              class="form-control"
+              [(ngModel)]="cell.newValue"
+              [name]="cell.getColumn().id"
+              [disabled]="!cell.getColumn().isEditable"
+              (click)="onClick($event)"
```

Comment:
> With the first comment, the thing is that at the moment looks like you are calling non-exising onClick(..) method
That's true! It is non existing in fact... I will fix that, thanks for pointing that out, I didn't noticed it.

>  thus we just asking you to make sure the code is building even with AOT
Great to know, thanks for pointing that out. I will!